app/actions/websocket.ts

```diff
@@ -641,17 +640,28 @@ function handleUserRemovedEvent(msg: WebSocketMessage) {
             const currentChannelId = getCurrentChannelId(state);
             const currentTeamId = getCurrentTeamId(state);
             const currentUser = getCurrentUser(state);
-            const actions: Array<GenericAction> = [{
-                type: ChannelTypes.CHANNEL_MEMBER_REMOVED,
-                data: {
-                    channel_id: msg.broadcast.channel_id,
-                    user_id: msg.data.user_id,
-                },
-            }];
+            const actions: Array<GenericAction> = [];
+            if (msg.data.user_id) {
+                actions.push({
+                    type: ChannelTypes.CHANNEL_MEMBER_REMOVED,
+                    data: {
+                        channel_id: msg.broadcast.channel_id,
+                        user_id: msg.data.user_id,
+                    },
+                });
+            } else if (msg.broadcast.user_id) {
+                actions.push({
+                    type: ChannelTypes.CHANNEL_MEMBER_REMOVED,
+                    data: {
+                        channel_id: msg.data.channel_id,
+                        user_id: msg.broadcast.user_id,
+                    },
+                });
```

Comment:
can we clean this code? maybe create a channelId and userId variable, assign depending on the condition and then just add the action to the array like

```js
const actions: Array<GenericAction> = [{
    type: ChannelTypes.CHANNEL_MEMBER_REMOVED,
    data: {
       channel_id: channelId,
       user_id: userId,
    }
}];
```

Reply:
done